org.openhab.binding.zigbee/ESH-INF/thing/channels.xml

```diff
@@ -146,6 +146,15 @@
         <state readOnly="true" />
     </channel-type>
 
+    <!-- IAS Tamper Channel -->
+    <channel-type id="ias_tamper">
+        <item-type>Switch</item-type>
+        <label>Tamper</label>
+        <description></description>
```

Comment:
Would it be sensible to add a description, like for most other channel types in this file?

Reply:
Could add `Indicates if a device is tampered`

Reply:
Small improvement: `Indicates if a device is tampered with`